packages/components/tsconfig.json

```diff
@@ -2,6 +2,9 @@
 	"extends": "../tsconfig",
 	"compilerOptions": {
 		"rootDir": "src",
-		"outDir": "build-module"
+		"outDir": "build-module",
+		"declaration": true,
+		"declarationMap": true,
+		"declarationDir": "./build-types",
```

Comment:
Since this is a JSON file, I think it's better to remove this trailing comma?

Reply:
Good catch 😄 Thanks!